src/index.js

```diff
@@ -3,24 +3,24 @@
 import chalk from 'chalk';
 import check from 'check-types';
 import program from 'commander';
-import promisify from 'es6-promisify';
 import inquirer from 'inquirer';
-import cmd from 'node-cmd';
 import path from 'path';
 import Rx from 'rx';
 import _package from '../package.json';
-import { QUESTION_TYPES } from './constants';
+import { PROMISIFIED_METHODS, QUESTION_TYPES } from './constants';
 import { store } from './createStore';
 import questions from './questions';
-import { postInstall } from './questions/index';
-import { checkForValidAppname, getQuestion, updateGenerator } from './services/Helpers';
+import { checkForValidAppname, updateGenerator } from './services/Helpers';
 import log from './services/log';
 import run from './services/run';
 
-const get = promisify(cmd.get, {
-    thisArg: cmd,
-    multiArgs: true,
-});
+/*
+* Some todos
+* */
+// TODO implement postInstall as a method on question classes. do this by adding all postInstall methods to an array and calling them afterwards.
```

Comment:
todo's liever in een markdown ipv in de code op allemaal verschillende plekken waar je ze weer vergeet